tests/unit_tests/dft/include/compute_inplace.hpp

```diff
@@ -58,12 +58,14 @@ int DFT_Test<precision, domain>::test_in_place_buffer() {
         }
         auto acc_host = inout_buf.template get_host_access();
         EXPECT_TRUE(check_equal_vector(acc_host.get_pointer(), out_host_ref_conjugate.data(),
-                                       inout_host.size(), abs_error_margin, rel_error_margin, std::cout));
+                                       inout_host.size(), abs_error_margin, rel_error_margin,
```

Comment:
Could you please merge the style changes for files that do not contain deprecated functions as separated commit? Otherwise it could cause some confusion what deprecation functions were changed in DFT files.

Reply:
Now only include the style changes caused by this PR in [0774034](https://github.com/oneapi-src/oneMKL/pull/287/commits/0774034224a726c5363cc9e301868fb1408f3a30).